jax/numpy/__init__.py

```diff
@@ -36,7 +36,7 @@
     fabs, finfo, fix, flatnonzero, flexible, flip, fliplr, flipud, float16, float32,
     float64, float_, float_power, floating, floor, floor_divide, fmax, fmin,
     fmod, frexp, full, full_like, function, gcd, geomspace, gradient, greater,
-    greater_equal, hamming, hanning, heaviside, histogram, histogram_bin_edges,
+    greater_equal, hamming, hanning, heaviside, histogramdd, histogram, histogram_bin_edges,
```

Comment:
minor: I think `histogramdd` should be after `histogram_bin_edges` to maintain sorted order.

Reply:
will do